engine/installation/linux/SUSE.md

```diff
@@ -1,112 +1,301 @@
+
+    pub  4096R/2C52609D 2015-07-14
+          Key fingerprint = 5811 8E89 F3A9 1289 7C07  0ADB F762 2157 2C52 609D
+    uid                            Docker Release Tool (releasedocker) <docker@docker.com>
+    ```
+
+    If the fingerprint matches, import the key:
+
+    ```bash
+    $ sudo rpm --import https://yum.dockerproject.org/gpg
+    ```
+
+2.  Use the following command to set up the **stable** repository:
+
+    ```bash
+    $ sudo zypper addrepo \
+        https://yum.dockerproject.org/repo/main/opensuse/13.2/ \
+        docker-main
+    ```
+
+3.  **Optional**: Enable the **testing** repository. You can enable it alongside
+    the stable repository. Do not use unstable repositories on on production
+    systems or for non-testing workloads.
+
+    > **Warning**: If you have both stable and unstable repositories enabled,
+    > updating without specifying a version in the `zypper install` or
+    > `zypper update` command will always install the highest possible version,
+    > which will almost certainly be an unstable one.
+
+
+    ```bash
+    $ sudo zypper addrepo \
+        https://yum.dockerproject.org/repo/testing/opensuse/13.2/ \
+        docker-testing
+    ```
+
+    You can disable a repository at any time by running the `zypper rmrepo`
+    command. The following command disables the `testing` repository.
+
+    ```bash
+    $ sudo zypper removerepo docker-testing
+    ```
+
+#### Install Docker
+
+1.  Update the `zypper` package index.
+
+    ```bash
+    $ sudo zypper refresh
+    ```
+
+2.  Install the latest version of Docker, or go to the next step to install a
+    specific version.
+
+    ```bash
+    $ sudo zypper install docker-engine
+    ```
+
+    > **Warning**: If you have both stable and unstable repositories enabled,
+    > installing or updating Docker without specifying a version in the
+    > `zypper install` or `zypper update` command will always install the highest
+    > available version, which will almost certainly be an unstable one.
+
+    The RPM will install, but you will receive the following error during the
+    post-installation procedure, because Docker cannot start the service
+    automatically:
+
+    ```none
+    Additional rpm output:
+    /var/tmp/rpm-tmp.YGySzA: line 1: fg: no job control
+    ```
+
+    Start Docker:
+
+    ```bash
+    $ sudo service docker start
+    ```
+
+3.  On production systems, you should install a specific version of Docker
+    instead of always using the latest. List the available versions. The
+    following example only lists binary packages and is truncated. To also list
+    source packages, omit the `-t package` flag from the command.
+
+    ```bash
+    $ zypper search -s --match-exact -t package docker-engine
+
+      Loading repository data...
+      Reading installed packages...
 
-## openSUSE
+      S | Name          | Type    | Version                               | Arch   | Repository    
```

Comment:
@mstanleyjones as previously suggested, can we abbreviate these listings and just show a couple of rows with elipsis? I don't think there's value in listing out everything that happened to be in there when this was written.

This applies to other distros too.